yt_idv/simple_gui.py

```diff
@@ -72,6 +74,12 @@ def render_camera(self, scene):
         if not imgui.tree_node("Camera"):
             return
         changed = False
+        _, scroll_delta = imgui.slider_float(
+            "Log Scroll Delta", math.log10(scene.camera.scroll_delta), -3.0, 0.0
+        )
```

Comment:
you have to drag the menu a pretty far ways to read the full "Log Scroll Data" label: 

![Screenshot from 2024-12-05 13-47-56](https://github.com/user-attachments/assets/5685fad9-ed99-4fed-a5c9-ad335754bb3d)

Making the slider narrower would make that easier to read. e.g.,:

```suggestion
        imgui.push_item_width(100)
        _, scroll_delta = imgui.slider_float(
            "Log Scroll Delta", math.log10(scene.camera.scroll_delta), -3.0, 0.0
        )
        imgui.pop_item_width()
```

Reply:
oh jeez, wow.  Should I put them on different lines?  Or use this?

Reply:
Oh, different lines would work too. If you use the push/pop item width, it'd look like 

![Screenshot from 2024-12-05 16-04-07](https://github.com/user-attachments/assets/c3aba831-9d21-4119-8f12-269cb1513fb6)

I don't have a strong preference on this versus splitting the label and slider to different lines, up to you!

Reply:
I used multiple lines after trying both.